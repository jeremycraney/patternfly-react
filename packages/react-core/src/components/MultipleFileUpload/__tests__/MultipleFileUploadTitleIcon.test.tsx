import React from 'react';
import { render, screen } from '@testing-library/react';
import { MultipleFileUploadTitleIcon } from '../MultipleFileUploadTitleIcon';

describe('MultipleFileUploadTitleIcon', () => {
  test('renders with expected class names', () => {
    const { asFragment } = render(<MultipleFileUploadTitleIcon data-testid="test-id" />);
    expect(asFragment()).toMatchSnapshot();
  });

  test('renders custom class names', () => {
    const { asFragment } = render(<MultipleFileUploadTitleIcon className="test" data-testid="test-id" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
