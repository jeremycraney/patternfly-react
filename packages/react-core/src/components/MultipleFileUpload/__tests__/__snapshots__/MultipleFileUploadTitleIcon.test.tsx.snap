// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MultipleFileUploadTitleIcon renders custom class names 1`] = `
<DocumentFragment>
  <div
    class="pf-c-multiple-file-upload__title-icon test"
    data-testid="test-id"
  />
</DocumentFragment>
`;

exports[`MultipleFileUploadTitleIcon renders with expected class names 1`] = `
<DocumentFragment>
  <div
    class="pf-c-multiple-file-upload__title-icon"
    data-testid="test-id"
  />
</DocumentFragment>
`;
